packages/internet_detector.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,37 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'internet_detector'
+$category = 'Utilities'
```

Comment:
Wondering if `Networking` is a better category than `Utilities`. @mandiant/flare-vm opinions?